config/locales/en.yml

```diff
@@ -204,6 +204,12 @@ en:
       header: "Study Cost Totals"
       grand_total: "Grand Total"
 
+  ###########
+  # CATALOG #
+  ###########
+  catalog:
```

Comment:
Can you move this to proper.en.yml? There's a section for catalog already in there.